app/src/main/java/ch/epfl/sdp/appart/Database.java

```diff
@@ -0,0 +1,10 @@
+package ch.epfl.sdp.appart;
+
+import java.util.List;
+
+public interface Database {
```

Comment:
Considering to change the name to `Repository` so that we can differentiate from the future local DB.